packages/contracts-bedrock/src/L1/L1ERC721Bridge.sol

```diff
@@ -23,19 +23,32 @@ contract L1ERC721Bridge is ERC721Bridge, ISemver {
     SuperchainConfig public superchainConfig;
 
     /// @notice Semantic version.
-    /// @custom:semver 2.0.0
-    string public constant version = "2.0.0";
+    /// @custom:semver 2.1.0
+    string public constant version = "2.1.0";
 
     /// @notice Constructs the L1ERC721Bridge contract.
-    /// @param _messenger   Address of the CrossDomainMessenger on this network.
-    constructor(address _messenger) ERC721Bridge(_messenger, Predeploys.L2_ERC721_BRIDGE) {
-        initialize(SuperchainConfig(address(0)));
+    constructor() ERC721Bridge() {
+        initialize({
+            _messenger: CrossDomainMessenger(address(0)),
+            _otherBridge: Predeploys.L2_ERC721_BRIDGE,
```

Comment:
We should set `_otherBridge` to `address(0)` to make the singleton completely broken